Content.Client/UserInterface/ActionMenu.cs

```diff
@@ -83,17 +83,17 @@ public ActionMenu(ClientActionsComponent actionsComponent, ActionsUI actionsUI)
                             {
                                 StyleClasses = { StyleNano.StyleClassActionSearchBox },
                                 HorizontalExpand = true,
-                                PlaceHolder = Loc.GetString("Search")
+                                PlaceHolder = Loc.GetString("generic-search")
```

Comment:
As `Search` is a verb, I'd argue some languages would have more or less appropriate verbs depending on the context.

(An example in english could be `Search` vs `Find`)

I'd actually like to relate this back to the generic `eject` entry from earlier, that probably suffers the same problem.
(as do the `Filter`, `Clear`, etc. below)

Please have a look at all generic entries you've added, and if they're verbs consider making them specific to the use case instead of generics (and if there then end up no uses of a specific generic, remove it)

Reply:
sure